packages/remix-react/package.json

```diff
@@ -14,6 +14,7 @@
   "main": "index.js",
   "module": "esm/index.js",
   "dependencies": {
+    "history": "^5.2.0",
```

Comment:
Is this just something that was missing? I thought we didn't need an explicit dep on `history` now...

Reply:
Whenever we import from a package in code it should be listed in deps regardless of if it's a dep of another dep we have listed. It helps bundlers and ESM CDN's like esm.sh. Without it more config is required to get remote modules to load. This is currently the only thing stopping us from consuming @remix-run packages from esm.sh.

Reply:
Yeah, I get that. I guess what I'm saying is let's remove whatever imports we're using from `history` in `remix-react` so we don't have to do this

Reply:
`parsePath` and `history` need to be re-exported before we can do this.